plugins/templates/create.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const hoek = require('hoek');
+const urlLib = require('url');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/templates',
```

Comment:
User should be able to do a `GET templates/{id}`, similar to `GET pipelines/{id}`

If they want to fetch by `name`, or `name+version`, they would use a query like `GET templates?name=node&version=123` or `GET templates?name=node`. It's more like a LIST. This is not implemented in this PR. It's the 3rd bullet point here (under using templates): https://github.com/screwdriver-cd/screwdriver/issues/470#issuecomment-285238348